client/tokenized-express-checkout/index.js

```diff
@@ -531,43 +532,35 @@ jQuery( ( $ ) => {
 				} );
 			} else {
 				// If this is the cart page, or checkout page, or pay-for-order page, we need to request the cart details.
-				getCartApiHandler()
-					.getCart()
-					.then( ( cartData ) => {
-						const total = transformPrice(
-							parseInt( cartData.totals.total_price, 10 ) -
-								parseInt(
-									cartData.totals.total_refund || 0,
-									10
-								),
-							cartData.totals
-						);
-						if ( total === 0 ) {
-							expressCheckoutButtonUi.hideContainer();
-							expressCheckoutButtonUi.getButtonSeparator().hide();
-						} else {
-							wcpayECE.startExpressCheckoutElement( {
-								mode: 'payment',
-								total,
-								currency: cartData.totals.currency_code.toLowerCase(),
-								// pay-for-order should never display the shipping selection.
-								requestShipping:
-									getExpressCheckoutData(
-										'button_context'
-									) !== 'pay_for_order' &&
-									cartData.needs_shipping,
-								shippingRates: transformCartDataForShippingRates(
-									cartData
-								),
-								requestPhone:
-									getExpressCheckoutData( 'checkout' )
-										?.needs_payer_phone ?? false,
-								displayItems: transformCartDataForDisplayItems(
-									cartData
-								),
-							} );
-						}
+				const cartData = await getCartApiHandler().getCart();
```

Comment:
This change introduced a difference in how errors are handled.

Before the change, any errors by `getCart()` weren't propagated, and they could have just generated a warning in the console.

Now, there is no `try`/`catch`, so any errors from `getCart()` are being propagated and will lead to a termination of the execution.

@danielmx-dev let me know your thoughts - I reverted this in https://github.com/Automattic/woocommerce-payments/pull/9849